src/plugins/ffe/ffe_database.py

```diff
@@ -126,21 +126,28 @@ def _generate_from_source_file(self, tmp_file: Path) -> bool:
         return True
 
     def _create_indexes(self):
-        self.write = True
-        with self:
-            self.execute(
-                'CREATE INDEX IF NOT EXISTS `player_last_name` ON `player`(`last_name` COLLATE NOCASE)'
-            )
-            self.execute(
-                'CREATE INDEX IF NOT EXISTS `player_first_name` ON `player`(`first_name` COLLATE NOCASE)'
-            )
-            self.execute(
-                'CREATE INDEX IF NOT EXISTS `player_fide_id` ON `player`(`fide_id`)'
-            )
-            self.execute(
-                'CREATE INDEX IF NOT EXISTS `player_ffe_licence` ON `player`(`ffe_licence_number` COLLATE NOCASE)'
+        try:
+            self.write = True
+            with self:
+                self.execute(
+                    'CREATE INDEX IF NOT EXISTS `player_last_name` ON `player`(`last_name` COLLATE NOCASE)'
+                )
+                self.execute(
+                    'CREATE INDEX IF NOT EXISTS `player_first_name` ON `player`(`first_name` COLLATE NOCASE)'
+                )
+                self.execute(
+                    'CREATE INDEX IF NOT EXISTS `player_fide_id` ON `player`(`fide_id`)'
+                )
+                self.execute(
+                    'CREATE INDEX IF NOT EXISTS `player_ffe_licence` ON `player`(`ffe_licence_number` COLLATE NOCASE)'
+                )
+                self.commit()
+        except Exception as e:
```

Comment:
```suggestion
        except (sqlite3.DatabaseError, sqlite3.OperationalError) as e:
```
is there a type of error you think might be outside of this ?